### PR TITLE
Update validation.php with 100 percent accuracy.

### PR DESCRIPTION
**Pull request by g-e-g-e-r-s-o-n**

### CODE REVIEW

File: src/ka/validation.php [hunk 1]
```diff
@@ -13,41 +14,41 @@
...
     'between'              => [
         'numeric' => ':attribute უნდა იყოს :min-სა და :max-ს შორის.',
         'file'    => ':attribute უნდა იყოს :min-სა და :max კილობაიტს შორის.',
         'string'  => ':attribute უნდა იყოს :min-სა და :max სიმბოლოს შორის.',
         'array'   => ':attribute-ის რაოდენობა უნდა იყოს :min-დან :max-მდე.',
     ],
-    'boolean'              => ':attribute უნდა იყოს true, false, 0 ან 1.',
-    'confirmed'            => ':attribute არ ემთხვევა დადასტურებას.',
+    'boolean'              => ':attribute ველი უნდა იყოს true ან false.',
+    'confirmed'            => ':attribute-ის დადასტურება არ ემთხვევა.',
```
Comment (caouecs) [inline src/ka/validation.php:33]: :attribute-ის it's working ?

File: src/ka/validation.php [hunk 2]
```diff
@@ -13,41 +14,41 @@
...
     'date'                 => ':attribute შეიცავს თარიღის არასწორ ფორმატს.',
-    'date_equals'          => ':attribute თარიღი უნდა იყოს :date.',
+    'date_equals'          => ':attribute უნდა იყოს :date-ის ტოლი თარიღი.',
```
Comment (caouecs) [inline src/ka/validation.php:35]: same here

File: src/ka/validation.php [hunk 3]
```diff
@@ -13,41 +14,41 @@
...
     'date_format'          => ':attribute არ ემთხვევა თარიღის ფორმატს: :format.',
     'different'            => ':attribute და :other არ უნდა ემთხვეოდეს ერთმანეთს.',
     'digits'               => ':attribute უნდა შედგებოდეს :digits ციფრისგან.',
     'digits_between'       => ':attribute უნდა შედგებოდეს :min-დან :max ციფრამბდე.',
     'dimensions'           => ':attribute შეიცავს სურათის არასწორ ზომებს.',
-    'distinct'             => ':attribute უნდა იყოს უნიკალური.',
+    'distinct'             => ':attribute-ის ველს აქვს დუბლირებული მნიშვნელობა.',
```
Comment (caouecs) [inline src/ka/validation.php:41]: same here

File: src/ka/validation.php [hunk 4]
```diff
@@ -79,43 +80,43 @@
         'numeric' => ':attribute არ უნდა აღემატებოდეს :max-ს.',
         'file'    => ':attribute არ უნდა აღემატებოდეს :max კილობაიტს.',
         'string'  => ':attribute არ უნდა აღემატებოდეს :max სიმბოლოს.',
-        'array'   => ':attribute-ის რაოდენობა არ უნდა აღემატებოდეს :max-ს.',
+        'array'   => ':attribute-ს არ უნდა ჰქონდეს :max ელემენტზე მეტი.',
```
Comment (caouecs) [inline src/ka/validation.php:83]: same here

File: src/ka/validation.php [hunk 5]
```diff
@@ -79,43 +80,43 @@
...
-    'mimes'                => ':attribute უნდა იყოს შემდეგი ტიპის: :values.',
-    'mimetypes'            => ':attribute უნდა იყოს შემდეგი ტიპის: :values.',
+    'mimes'                => ':attribute უნდა იყოს შემდეგი ფაილური ტიპის: :values.',
+    'mimetypes'            => ':attribute უნდა იყოს შემდეგი ფაილური ტიპის: :values.',
     'min'                  => [
         'numeric' => ':attribute უნდა იყოს მინიმუმ :min.',
         'file'    => ':attribute უნდა იყოს მინიმუმ :min კილობაიტი.',
         'string'  => ':attribute უნდა შეიცავდეს მინიმუმ :min სიმბოლოს.',
-        'array'   => ':attribute უნდა იყოს მინიმუმ :min.',
+        'array'   => ':attribute-ს უნდა ჰქონდეს მინიმუმ :min ელემენტი.',
```
Comment (caouecs) [inline src/ka/validation.php:91]: same

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you, but I don't know for the modification of tags. just confirm before

File: src/ka/validation.php (hunk 1 above)
Comment (g-e-g-e-r-s-o-n) [reply src/ka/validation.php:33]: yes, of course.

regex

Comment (g-e-g-e-r-s-o-n) [reply src/ka/validation.php:33]: I tested it on regex and :attribute-ის is correct  and it's working properly.

File: src/ka/validation.php (hunk 2 above)
Comment (g-e-g-e-r-s-o-n) [reply src/ka/validation.php:35]: it's working properly.

File: src/ka/validation.php (hunk 3 above)
Comment (g-e-g-e-r-s-o-n) [reply src/ka/validation.php:41]: it's working properly.

File: src/ka/validation.php (hunk 4 above)
Comment (g-e-g-e-r-s-o-n) [reply src/ka/validation.php:83]: it's working properly

File: src/ka/validation.php (hunk 5 above)
Comment (g-e-g-e-r-s-o-n) [reply src/ka/validation.php:91]: it's working properly

**Comment by caouecs** [comment]
Thank you

**Pull request merged**